phy/cluster/manual/session.py

```diff
@@ -201,18 +188,36 @@ def _store_cluster(self,
                               append=True,
                               )
 
-    def store_all_clusters(self, spikes_per_cluster):
+    def is_consistent(self, cluster, spikes):
+        """Return whether the data of a cluster is consistent."""
```

Comment:
> In fairness this is probably more "Return whether the filesize of a cluster store file is correct" - you're not checking the data at all, even less for consistency!

on the other hand, I feel that just checking the file size is sufficient in 99.9999% of the cases. Why? Because:

* We check for consistency of *all* files: if a single file doesn't have the right size, the cluster store is declared inconsistent
* Cluster ids are unique: when you change a cluster, its identifier changes. So if you have two cluster stores with the same number of files, the same ids, and the same number of bytes in every file, I can hardly see how the clusterings can be different...